src/javascript/binary/base/page.js

```diff
@@ -162,12 +162,8 @@ Page.prototype = {
         }
     },
     general_authentication_message: function() {
-        const div = $('<div/>', { text: localize('Please send us the following documents in order to verify your identity and authenticate your account:') });
```

Comment:
@jy-binary , that was the instructions I've got :) Just forwarded you email conversation on this topic.